vmdb/config/vmdb.tmpl.yml

```diff
@@ -288,6 +288,16 @@ workers:
         :duration: 10.seconds
         :capacity: 50
         :amqp_port: 5672
+      :event_catcher_openstack_infra:
+        :poll: 15.seconds
+        :topics:
+          :nova: "notifications.*"
+          :cinder: "notifications.*"
+          :glance: "notifications.*"
+          :quantum: "notifications.*"
+        :duration: 10.seconds
+        :capacity: 50
+        :amqp_port: 5672
```

Comment:
If EventCatcherOpenstackInfra is a subclass of EventCatcherOpenstack, then this needs to be indented under the : event_catcher_openstack key.  Additionally, since values are then inherited, you shouldn't need to specify any values for :event_catcher_openstack_infra.